### PR TITLE
added some leagues to the sports/football api

### PR DESCRIPTION
**Pull request by Eiiki**
Now apis provides an api to the top football leagues in Icelandic football.

### CODE REVIEW

**Comment by koddsson** [comment]
Hey @Eiiki, thanks for the patch! I really appreciate when people donate their time and effort to make apis better :)

One thing I'm wondering about the patch is if it would be better to deal with the league filtering using query parameters rather than URL paramaters so the pepsi league would be changed from `/sports/football/male-leagues/pepsi` to something like `/sports/football/male-leagues?league=pepsi`. It would be more [DRY](http://en.wikipedia.org/wiki/Don't_repeat_yourself) as well as it would not need any code change once the leagues change name (I'm assuming they do since they are all named after some companies :)

How does that sound?

File: endpoints/sports/index.js [hunk 1]
```diff
@@ -37,7 +37,50 @@ app.get('/sports', function (req, res, next) {
...
+});
+
+/* Root footbal female leagues handler */
+app.get('/sports/football/female-leagues', function (req, res, next) {
+
+  return res.json(
+    {
+      results: [
+        {
+          info: 'This is an api for Ielandic female football leagues',
```
Comment (koddsson) [inline endpoints/sports/index.js]: Icelandic* ;)

**Comment by koddsson** [comment]
This looks okay to me. Maybe you can add some tests for this @Eiiki?

What do you think @MiniGod, @kristjanmik and @benediktvaldez?

**Comment by Eiiki** [comment]
Successfull tests added.

**Comment by koddsson** [comment]
I just pulled and ran the tests locally

```
☁  apis [sports-football] npm test

> apis.is@0.3.0 test /Users/koddsson/Repos/apis
> mocha test/integration

child_process: customFds option is deprecated, use stdio instead.

  ․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․

  84 passing (19s)
  7 pending
```

I also inspected some data coming from some of the endpoints and they looked OK, barring some encoding issues that I don't think are a major problem and if they turn out to be problems people can always open up issues or better yet PRs :)

With all that in mind, I'm merging this!

**Comment by koddsson** [comment]
@Eiiki You're now a apis contributor :tada:

**Comment by Eiiki** [comment]
woooo!

**Pull request merged**